pkg/planner/property/physical_property.go

```diff
@@ -253,6 +255,9 @@ type PhysicalProperty struct {
 		*expression.VSInfo
 		TopK uint32
 	}
+
+	// IsParentPhyscicalHashJoin indicates whether the parent of the current operator is a physical join.
+	IsParentPhyscicalHashJoin bool
```

Comment:
do we really need this?

Reply:
If we don't consider it, ```TestMPPJoinWithoutRemoveUselessExchange``` will fail and get wrong message.